<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAMProfileSettings</class>
 <widget class="QDialog" name="CAMProfileSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Options</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>12</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>&lt;p align=&quot;right&quot;&gt;Name:&lt;/p&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leOPName"/>
     </item>
     <item>
      <widget class="QPushButton" name="bFindProfileOperation">
       <property name="text">
        <string>Find Profile Operation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_3">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Savety margin:&lt;/p&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>leWidth</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leSavetyMargin">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Z Retract:&lt;/ps&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leZRetract">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_2">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Target Depth:&lt;/p&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leTargetDepth">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_2_3">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Depth Increment:&lt;/p&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leDepthIncrement">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="title">
      <string>Spindle</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_2_2_2_2_3">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;RPM:&lt;/ps&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leSpindleRPM">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="rbSPCW">
          <property name="text">
           <string>CW</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbSPCCW">
          <property name="text">
           <string>CCW</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QGroupBox" name="buttonGroup2">
     <property name="title">
      <string>Cutter Compensation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_3">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Tool Diam:&lt;/p&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leToolDiameter">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_3_2">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Tool No:&lt;/p&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leToolNumber">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>Feed Rates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_2_2_2_2_2">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Cut:&lt;/p&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leFeedCutRate">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_2_2_2_2">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Plunge:&lt;/p&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leFeedPlungeRate">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>Optional</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_2_2">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;First Increment:&lt;/p&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leFirstIncrement">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2_2_2_2">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Last Increment:&lt;/p&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leLastIncrement">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bOk">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leOPName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSavetyMargin</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leZRetract</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTargetDepth</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leDepthIncrement</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFeedCutRate</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFeedPlungeRate</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSpindleRPM</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leToolDiameter</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leToolNumber</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFirstIncrement</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLastIncrement</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAMProfileSettings</receiver>
   <slot>testForm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>testForm(QString)</slot>
 </slots>
</ui>
